ui/ui-components/components/navigation/setupStepsNavProgressBar.tsx

```diff
@@ -11,6 +12,9 @@ export default function SetupStepsNavProgressBar({
 }) {
   const [steps, setSteps] = useState<Models.SetupStepType[]>([]);
   const [shouldDisplay, setShouldDisplay] = useState(false);
+  const [initialOnBoardingState, setInitialOnBoardingState] = useState(null);
+  //because shouldDisplay is set on every Setup Step call, track if a user manually hides setup steps separately
+  const [hideCard, setHideCard] = useState(null);
```

Comment:
Typescript thing: If you set an initial `useState` as `null`, Typescript can't guess what the type of the value should be.  You can use `<>` to set a type specifically, in this case:

```
const [hideCard, setHideCard] = useState<boolean>(null);
```

So only `true,false,null` would be allowed

Reply:
Nice!  And that will evaluate to `null`, not false?

Reply:
The `<boolean>` doesn't change the /value/ of any JS code - it's just telling Typescrip that the type of `hideCard` is boolean, so you'll get compile-time errors if you did something like `hideCard ='fish'`